docker-compose.yml

```diff
@@ -1,28 +1,33 @@
 version: '2'
 services:
-#Please set the environment variables before start running the container with docker compose.
-   db_bot:
+#Please set the environment variables in env file in below mentioned format before start running the container with docker compose.
+#MYSQL_ROOT_PASSWORD=mysql_root_pass
+#SLACK_BOT_TOKEN=bot_token
+#CHATBOT_NAME=bot_name
+#APPROVER_SLACK_NAME=approver_slack_id
+#JENKINS_URL=jenkins_url
+#JENKINS_USER=jenkins_user_name
+#JENKINS_PASS=jenkins_user_pass
+
+   db:
      container_name: db_jenkins_bot
      build:
        context: https://github.com/USDevOps/jenkins_slack_bot.git
        dockerfile: Dockerfile
-     environment:
-       MYSQL_ROOT_PASSWORD : "MYSQL_PASS"
-       SLACK_BOT_TOKEN : "BOT_TOKEN"
-       CHATBOT_NAME : "BOT_NAME"
-       APPROVER_SLACK_NAME : "APPROVER_SLACK_ID"
+     env_file:
+        - ./vars.env
      restart: always
 
-   jenkins_bot:
-     container_name: jenkins_slack_bot
-     extends: db_bot
+   jenkins_bot_rtm:
+     container_name: jenkins_slack_bot_rtm
+     extends: db
      entrypoint: python3.4 slackbot.py
```

Comment:
Do we need to configure restart policy?

Reply:
I think restart policy is in version 3 not in version 2 . This file is written in version 2.

Reply:
@vivekgrover1 , checking v2, restart is supported.
https://docs.docker.com/compose/compose-file/compose-file-v2/#restart

Could you double confirm this?

If v2 really doesn't support it, maybe we should remove L19?

Reply:
@DennyZhang I have included the restart in all the services.